### PR TITLE
feat: add `--include-token` flag to `okteto ctx show` command

### LINKED ISSUE #3935: Remove `token` from `okteto context show` by default
**Is your feature request related to a problem? Please describe.**
When executing the `okteto context show` command the object returns the `token` value, which is not best practice for security.

**Describe the solution you'd like**
We should remove the `token` value by default from this object.

We should also consider adding an option `--include-token` to the `okteto context show` command to optionally include that token in scenarios where it's useful. This way the disclosure of the `token` value can be better controlled/managed.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by nitishfy**
# Proposed changes

Fixes #3935

After changes:

image

### CODE REVIEW

**Comment by nitishfy** [comment]
There is no test file for `show.go`, Should we try adding test cases for it?  Maybe we can write tests about output validation.
cc: @jLopezbarb @andreafalzetti

File: cmd/context/show.go [hunk 1]
```diff
@@ -43,6 +44,18 @@ func Show() *cobra.Command {
...
+			TokenValue := current.Token
+
+			// By default, set the token value to null since default behaviour doesn't require to show token value
+			current.Token = ""
+
+			// If the flag is set, display the token value
+			if includeToken {
+				current.Token = TokenValue
+			}
+
```
Comment (andreafalzetti) [inline cmd/context/show.go:58]: Why not simplify like this?

```suggestion
			if !includeToken {
				current.Token = ""
			}

```

File: cmd/context/show.go [hunk 2]
```diff
@@ -62,7 +75,7 @@ func Show() *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVarP(&output, "output", "o", "json", "output format. One of: ['json', 'yaml']")
-
+	cmd.Flags().BoolVar(&includeToken, "include-token", false, "show the token value to the user")
```
Comment (andreafalzetti) [inline cmd/context/show.go:78]:
```suggestion
	cmd.Flags().BoolVar(&includeToken, "include-token", false, "include the token in the output")
```

**Comment by jLopezbarb** [comment]
> There is no test file for `show.go`, Should we try adding test cases for it? Maybe we can write tests about output validation. cc: @jLopezbarb @andreafalzetti

If you want to give it a try it would be awesome!! If you face any problem let us know and we will be willing to help you

File: cmd/context/show.go (hunk 1 above)
Comment (AdrianPedriza) [reply cmd/context/show.go:58]: I would use this. Looks clearer

**Comment by AdrianPedriza** [comment]
we need an info log about use `--include-token` when is not set. https://github.com/okteto/okteto/issues/3935#issuecomment-1742786094

**Comment by andreafalzetti** [comment]
> we need an info log about use `--include-token` when is not set. [#3935 (comment)](https://github.com/okteto/okteto/issues/3935#issuecomment-1742786094)

What do you mean? In `okteto context show` the output is JSON or YAML so a log would break it

File: cmd/context/show.go (hunk 1 above)
Comment (andreafalzetti) [reply cmd/context/show.go:58]: hey @nitishfy, can you apply this change? wdyt?

File: cmd/context/show.go (hunk 2 above)
Comment (nitishfy) [reply cmd/context/show.go]: Resolved

File: cmd/context/show.go (hunk 1 above)
Comment (nitishfy) [reply cmd/context/show.go]: Resolved

File: cmd/context/show.go [hunk 3]
```diff
@@ -62,8 +68,7 @@ func Show() *cobra.Command {
 		},
 	}
 	cmd.Flags().StringVarP(&output, "output", "o", "json", "output format. One of: ['json', 'yaml']")
-
-	return cmd
+    cmd.Flags().BoolVar(&includeToken, "include-token", false, "include the token in the output")	return cmd
```
Comment (AdrianPedriza) [inline cmd/context/show.go:71]: a bad format here. does not compile
```suggestion
    cmd.Flags().BoolVar(&includeToken, "include-token", false, "include the token in the output")
    return cmd
```

Comment (nitishfy) [reply cmd/context/show.go]: Sorry! Resolved now

**Review by AdrianPedriza — APPROVED** [review]

**Pull request merged**